### DISCUSSION TITLE
Welcome to WEATS Discussions!

### DISCUSSION THREAD

**Discussion by mcgeeyoung** [wattcarbon/open-eac-alliance#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thomascwells** [comment]
@mcgeeyoung / @ssuffian - I think getting folks engaged in the discussions here would be a great way to keep some community momentum going between meetings.
